### ISSUE TITLE
Sundials installation issue on Windows -- unresolved externals

### ISSUE THREAD

**Issue by RiddlemanMVP** [LLNL/sundials#347]
Hello,

I'm experiencing some issues during the installation of the Sundials libraries on Windows 10.

I followed the steps described in the [documentation](https://sundials.readthedocs.io/en/latest/Install_link.html#configuring-building-and-installing-on-windows). I went on and installed CMake 3.27.6 and I'm running Visual Studo 17.7.2.

Step 1 to 4 run well. 
Step 5a produces a total of 3687 errors (all of them seem to be LNK1120 -- unresolved externals)
Step 5b produces the same result. The installation directory INSTDIR is empty.

I believe this must have something to do with Visual Studio's linker...
does anyone have an idea how I solve this?

**Comment by balos1** [comment]
Did you build with static libraries enabled? Those may not work currently with Visual Studio. Try adding `-DBUILD_STATIC_LIBS=OFF` to your CMake command.

**Comment by mvsivaselvan** [comment]
I realize it has been a while, so likely a resolution has been found. But in case it is useful, I ran into a similar issue, and posted my resolution in the Google group under the subject "IDA installation on Windows". 

For the unresolved externals linker error that I got, I had to only build the static library and not the DLL using the option -DBUILD_SHARED_LIBS=NO when calling cmake. I also wrote my reasoning in the Google group post.

Hope this is useful.
